### ISSUE TITLE
Update task causes workbench --check to crash

### ISSUE THREAD

**Issue by Gii2000** [mjordan/islandora_workbench#474]
Running `python3 workbench --config config.yml --check` crashes on an `update` task with the `replace` update mode. (It runs successfully without the `--check` flag.)
```
Traceback (most recent call last):
  File "~/islandora_workbench/workbench", line 1285, in <module>
    check_input(config, args)
  File "~/islandora_workbench/workbench_utils.py", line 1757, in check_input
    if len(rows_with_missing_files) > 0 and config['strict_check'] is False:
UnboundLocalError: local variable 'rows_with_missing_files' referenced before assignment
```

**Comment by mjordan** [comment]
No doubt a regression error introduced in 51c0f79d1683104aa262e0431ebb411f535f0243. I'll fix. Thanks for reporting it.

**Comment by mjordan** [comment]
Resolved with b6f1394dcc8c58351e1f428e77507da9a49b336d.
